web/src/filter.ts

```diff
@@ -1047,6 +1047,17 @@ export class Filter {
         return this._terms.length === 1 && this.has_operand("in", "home");
     }
 
+    is_in_channel_topic_narrow(): boolean {
+        if (
+            this.terms().length === 2 &&
+            this.has_operator("channel") &&
+            this.has_operator("topic")
+        ) {
+            return true;
+        }
+        return false;
+    }
```

Comment:
Why do you need this new function? I feel like it feels very similar to the ones about `is_conversation_view`, for example, and I'm not sure what the goal is. Do you want just `is_channel_conversation_view`, or something else?